buildSrc/src/main/kotlin/partiql.versions.kt

```diff
@@ -53,6 +53,8 @@ object Versions {
     const val junit4 = "4.12"
     const val junit4Params = "1.1.1"
     const val mockito = "4.5.0"
+    const val snapshotTest = "4.0.8"
+    const val slf4j = "2.0.17"
```

Comment:
Thanks for going about adding the snapshot tests. Can we remove this slf4j dependency? I didn't need to use it in the other PR https://github.com/partiql/partiql-graphviz-java/pull/11.